packages/debug/src/browser/debug-frontend-application-contribution.ts

```diff
@@ -721,31 +717,9 @@ export class DebugFrontendApplicationContribution extends AbstractViewContributi
         });
         registry.registerCommand(DebugSessionContextCommands.REVEAL, {
             execute: () => this.selectedSession && this.revealSession(this.selectedSession),
-            isEnabled: () => this.hasSessionWidget,
-            isVisible: () => !this.selectedThread && this.hasSessionWidget
-        });
-        registry.registerCommand(DebugSessionContextCommands.OPEN_LEFT, {
```

Comment:
Suggestion: Should we mark `OPEN_LEFT`, `OPEN_RIGHT` and `OPEN_BOTTOM` as deprecated? They don't have an implementation anymore. They should probably also be removed from this code:

https://github.com/eclipse-theia/theia/blob/0512a5f4ede37547a9949d514936daf8c972fcb8/packages/debug/src/browser/debug-frontend-application-contribution.ts#L559-L564